sunpy/map/mapbase.py

```diff
@@ -566,15 +566,25 @@ def _get_cmap_name(self):
         cmap_string = self.observatory + self.meta['detector'] + str(int(self.wavelength.to('angstrom').value))
         return cmap_string.lower()
 
-    def _validate(self):
+    def _validate_meta(self):
         """Validates the meta-information associated with a Map.
 
-        This function includes very basic validation checks which apply to
+        This method includes very basic validation checks which apply to
         all of the kinds of files that SunPy can read. Datasource-specific
         validation should be handled in the relevant file in the
-        sunpy.map.sources package."""
+        sunpy.map.sources package.
+
+        Allows for default unit assignment for:
+            CUNIT1, CUNIT2, WAVEUNIT,  """
 #        if (self.dsun <= 0 or self.dsun >= 40 * constants.au):
 #            raise InvalidHeaderInformation("Invalid value for DSUN")
+
+        warnings.simplefilter('always', Warning)
+
+        for meta_property in ('cunit1', 'cunit2', 'waveunit'):
+            if (self.meta.get(meta_property) != None) and (u.Unit(self.meta.get(meta_property), parse_strict = 'silent').physical_type == 'unknown'):
```

Comment:
you do not need the `!= None` here, just `if self.meta.get(meta_property) and ...` will work. (None evaluates to False)